### ISSUE TITLE
Create ServiceCardComponent

### ISSUE THREAD

**Issue by saperez17** [saperez17/Carfy#7]
Create a React component to display services available. The component should render something like the image below. The card should include: 
-title
-price
-services (this is a string list like ['oil change', 'maintenance']
-description.
![image](https://user-images.githubusercontent.com/34078372/113522691-4dd6eb00-9568-11eb-9158-abacd8926ef0.png)

**Comment by philipep-galdino** [comment]
Wouldn't it be better if home service wasn't really part of the description, but an option?

The service provider could tell us if that service is available to be done at home, something like this:

Home service: available
Home service: unavailable

**Comment by saperez17** [comment]
> Wouldn't it be better if home service wasn't really part of the description, but an option?
> 
> The service provider could tell us if that service is available to be done at home, something like this:
> 
> Home service: available
> Home service: unavailable

That's right. The "home" in the text description is just some sample text I put there. That whole text should be taken from the description attribute of e shop_service instance .
